Features/CaseSearch/test_pages/casesearch_page.py

```diff
@@ -7,7 +7,8 @@
 from selenium.webdriver.common.by import By
 
 from common_utilities.selenium.base_page import BasePage
-from Features.CaseSearch.constants import TEXT_INPUT, COMBOBOX, YES, NO, text, combobox, HOME, WORK, PREV_MENU, MENU, FIRST_MENU, HOME_SCREEN
+from Features.CaseSearch.constants import TEXT_INPUT, COMBOBOX, YES, NO, text, combobox, HOME, WORK, PREV_MENU, MENU, \
```

Comment:
try to import Features.CaseSearch.constants as a whole instead of importing each one.

Reply:
I can import all using `Features.CaseSearch.constants import *`, but is there anything concerning with this? @kbo001

Reply:
No, nothing concerning. It just the import looks really long. So you can simply write `import Features.CaseSearch.constants`. If it gives any errors then just go with the original. Other than that all looks good to me.

Reply:
` import Features.CaseSearch.constants` doesn't work, so added `from Features.CaseSearch.constants import *` in
https://github.com/dimagi/dimagi-qa/pull/171/commits/4419835dbb442ad22e320b577dda7514b1b1a1d9